geomesa-hbase/geomesa-hbase-rpc/src/main/scala/org/locationtech/geomesa/hbase/rpc/filter/Z3HBaseFilter.scala

```diff
@@ -6,7 +6,7 @@
  * http://www.opensource.org/licenses/apache2.0.php.
  ***********************************************************************/
 
-package org.locationtech.geomesa.hbase.filters
+package org.locationtech.geomesa.hbase.rpc.filter
```

Comment:
What's the goal of the 'rpc' package?  Some of the stuff makes a lot of sense there.  Is there any reason to toss the HBase filters in that package?

Reply:
the rpc module is the bridge between client (datastore) and server modules. i think everything in there is needed by both... i was trying to make each module only contain a single top-level package for future java 11 support.